Corpo briefcases

Davyei opened the pull request:
<!--
You are making this pull request for the Moffstation fork of Space Station 14.

Please be sure to follow general guidelines for upstream PRs, but also be sure to follow the Moffstation guidelines.
Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline
See the Harmony contributing guidelines for an example on what we want: https://github.com/ss14-harmony/ss14-harmony/blob/master/CONTRIBUTING.md
-->

## About the PR
<!-- What did you change? -->
Adds four new corporate briefcases sprited by Kento_da_neko (discord)
## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
cool briefcases
## Technical details
<!-- Summary of code changes for easier review. -->

## Media
<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc).
Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->
<img width="387" height="337" alt="Screenshot 2025-10-10 152610" src="https://github.com/user-attachments/assets/fa415d7b-6f55-4f7a-adfb-0373918454d4" />
<img width="530" height="385" alt="Screenshot 2025-10-10 152228" src="https://github.com/user-attachments/assets/dacf53b8-689e-4d43-a6c2-99574fae426b" />
<img width="578" height="386" alt="Screenshot 2025-10-10 152203" src="https://github.com/user-attachments/assets/b2970bcf-d88b-4761-88f8-ee48e378b5fe" />
<img width="368" height="112" alt="Screenshot 2025-10-10 141447" src="https://github.com/user-attachments/assets/ca2e0593-766b-4850-a6fd-b9208488de0a" />
<img width="336" height="380" alt="Screenshot 2025-10-13 133952" src="https://github.com/user-attachments/assets/4230d1cd-4aa7-43db-987e-5d7a7009ba34" />

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X ] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X ] I have properly sectioned my changes into fork namespaces, and/or followed proper guidelines for modifying upstream files.
- [X ] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them.
This will be posted in #codebase-changes. -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
<!--
:cl:
- add: Added fun!
- remove: Removed fun!
- tweak: Changed fun!
- fix: Fixed fun!
-->
:cl:
- add; four new Corporate Briefcases.
- tweak: lobbying bundle & sniper bundle are now syndicate-themed briefcases.

Davyei committed:
Corporate Briefcases have been added

Adds the NT & Syndicate briefcases sprited by @kento_da_neko (discord)

the sniper bundle & lobbying bundle now use the syndicate sprites and the NanoTrasen professional & propaganda briefcase fills have been added too.

Davyei committed:
removes non-existant id

the nanocard isn't real woopsie daisy

Centronias reviewed (dismissed):
I'm confused by the sizes being changed in a few places in the inheritance hierarchy, but it looks like that was already a thing on upstream, so idk.

Davyei committed:
Update Resources/Prototypes/_Moffstation/Entities/Objects/Misc/briefcases.yml

Co-authored-by: Centronias <charlie.t.santos@gmail.com>
Signed-off-by: Dave <davidaperry2@gmail.com>

Davyei committed:
Update Resources/Prototypes/_Moffstation/Entities/Objects/Misc/briefcases.yml

Co-authored-by: Centronias <charlie.t.santos@gmail.com>
Signed-off-by: Dave <davidaperry2@gmail.com>

Davyei committed:
Update Resources/Prototypes/Catalog/Fills/Items/briefcases.yml

Co-authored-by: Centronias <charlie.t.santos@gmail.com>
Signed-off-by: Dave <davidaperry2@gmail.com>

Davyei committed:
Business Negotiation conclusions (fixes)

Centronias's fixes plus some new sprites Kento sent me for a centcomm briefcase.

southbridge-fur committed:
Update Resources/Prototypes/_Moffstation/Entities/Objects/Misc/briefcases.yml

Co-authored-by: Centronias <charlie.t.santos@gmail.com>
Signed-off-by: Southbridge <7013162+southbridge-fur@users.noreply.github.com>

southbridge-fur committed:
Update Resources/Prototypes/Entities/Objects/Misc/briefcases.yml

Co-authored-by: Centronias <charlie.t.santos@gmail.com>
Signed-off-by: Southbridge <7013162+southbridge-fur@users.noreply.github.com>

southbridge-fur committed:
Merge branch 'master' of https://github.com/moff-station/moff-station-14 into fork/Davyei/Corpo-Briefcases

southbridge-fur commented:
The checks are being weird

southbridge-fur committed:
Empty commit

Centronias commented on Resources/Prototypes/_Moffstation/Catalog/Fills/Items/briefcases.yml:
```diff
@@ -0,0 +1,47 @@
+
+- type: entity
+  id: BriefcaseCentcomm
+  parent: BriefcaseCentcomm
```
```suggestion
  id: BriefcaseCentcomm
  parent: BriefcaseBase
```
I'm not sure if this is the correct parent, but inheriting from itself is perhaps the issue.

Centronias committed:
Update Resources/Prototypes/_Moffstation/Catalog/Fills/Items/briefcases.yml

Signed-off-by: Centronias <charlie.t.santos@gmail.com>

Centronias committed:
Update Resources/Prototypes/_Moffstation/Catalog/Fills/Items/briefcases.yml

github web ui gaming

Signed-off-by: Centronias <charlie.t.santos@gmail.com>

Centronias committed:
Update Resources/Prototypes/_Moffstation/Catalog/Fills/Items/briefcases.yml

Signed-off-by: Centronias <charlie.t.santos@gmail.com>
